specs/Workers.md

```diff
@@ -0,0 +1,1389 @@
+Workers support in WebView2
+===
+
+# Background
+Currently, WebView2 lacks comprehensive APIs for developers to fully utilize
```

Comment:
Would be good to add a note to the background that this API spec covers some basics for workers and that future API specs will add additional worker features (with examples & links to the other API specs if available)